README.md

```diff
@@ -524,6 +524,8 @@ There are several global variables you can use to introspect on Bats tests:
 * `$BATS_TEST_DESCRIPTION` is the description of the current test case.
 * `$BATS_TEST_NUMBER` is the (1-based) index of the current test case in the
   test file.
+* `$BATS_GLOBAL_TEST_NUMBER` is the (1-based) index of the current test case in
+ the test run (over all files).
```

Comment:
I'd suggest using the term "test suite" here.

Reply:
You forgot to change "the current test case in the test run" to use "test suite" here.

Reply:
Thx, I only changed the manpage.